wire-ios/Wire-iOS/Sources/UserInterface/Settings/Backup/BackupPasswordViewController.swift

```diff
@@ -71,24 +72,40 @@ final class BackupPasswordViewController: UIViewController {
     }
 
     private func setupViews() {
-        view.backgroundColor = SemanticColors.View.backgroundDefault
+        view.backgroundColor = ViewColors.backgroundDefault
         passwordRulesLabel.text = PasswordRuleSet.localizedErrorMessage
 
         [passwordView, subtitleLabel, passwordRulesLabel].forEach {
             view.addSubview($0)
             $0.translatesAutoresizingMaskIntoConstraints = false
         }
 
-        passwordView.placeholder = HistoryBackup.Password.placeholder.capitalized
+        passwordView.placeholder = HistoryBackup.Password.placeholder
         passwordView.accessibilityIdentifier = "password input"
         passwordView.accessibilityHint = PasswordRuleSet.localizedErrorMessage
         passwordView.returnKeyType = .done
         passwordView.isSecureTextEntry = true
         passwordView.delegate = self
         passwordView.textColor = LabelColors.textSectionHeader
         passwordView.backgroundColor = ViewColors.backgroundUserCell
-        let attributes: [NSAttributedString.Key: Any] = [.foregroundColor: SemanticColors.SearchBar.textInputViewPlaceholder,
-                                                        .font: FontSpec.smallRegularFont.font!]
+
+        // Set a fixed font size
+        let fixedFont = UIFont.systemFont(ofSize: 17)
+        passwordView.font = fixedFont
+
+        // Enable large content viewer
+        passwordView.showsLargeContentViewer = true
+
+        // Customize large content viewer
+        passwordView.largeContentTitle = HistoryBackup.Password.placeholder.capitalized
```

Comment:
Out of interest, why do you capitalize this but not the `placeholder`?

Reply:
This has been reverted. https://github.com/wireapp/wire-ios/commit/7498791dd2b1b5001260e26e8e12e5de828038ad